### ISSUE TITLE
Amazon GuardDuty supports exporting findings to an Amazon S3 bucket

### ISSUE THREAD

**Issue by ewbankkit** [terraform-providers/terraform-provider-aws#10920]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Amazon GuardDuty customers can now export findings to Amazon S3.

### New or Affected Resource(s)

* `aws_guardduty_publishing_destination`

### Potential Terraform Configuration

```hcl
resource "aws_guardduty_publishing_destination" "example" {}
```

### References

[Announcement](https://aws.amazon.com/about-aws/whats-new/2019/11/amazon-guardduty-supports-exporting-findings-to-an-amazon-s3-bucket/).
[User guide](https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_exportfindings.html).
[API reference](https://docs.aws.amazon.com/guardduty/latest/APIReference/API_CreatePublishingDestination.html)

**Comment by rverma-jm** [comment]
Any updates on this?

**Comment by Abhisek1994Roy** [comment]
Is there a sample aws template to enable this?

**Comment by harshalx** [comment]
Doesn't this resource fulfill the same functionality ?
https://www.terraform.io/docs/providers/aws/r/guardduty_threatintelset.html

**Comment by harshalx** [comment]
Any updates on this? We need this desperately.

**Comment by bendehaan** [comment]
@shaepe and I have cooked up a PR for this some time ago: https://github.com/terraform-providers/terraform-provider-aws/pull/13894. Could anyone review?

**Comment by bendehaan** [comment]
PR https://github.com/terraform-providers/terraform-provider-aws/pull/13894/ is approved. Any timeline for when it will be merged?

**Comment by bflad** [comment]
Support for this new resource has been merged and will release with version 3.4.0 of the Terraform AWS Provider, later this week. Thanks to @bendehaan and @shaepe for the implementation. 👍
